src/theme/components/StatusBage.tsx

```diff
@@ -0,0 +1,25 @@
+import { tagAnatomy } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(tagAnatomy.keys)
+
+const vocdoniStatusBadge = definePartsStyle({
+  container: {
+    padding: '4px 12px',
+    borderRadius: '4px',
+    height: '18px',
+    fontStyle: 'normal',
+    fontWeight: '700',
+    fontFamily: 'Manrope',
```

Comment:
Always define fallback fonts.

Reply:
I delete the fontFamily attribute because is inherited from the old explorer and is not actually needed for the moment. On a near future we probably will style the explorer adding custom fonts, so a this moment is not needed to define this... 

d912f96